feature/communitylist/src/main/java/com/iteneum/communitylist/presentation/CommunityListView.kt

```diff
@@ -0,0 +1,41 @@
+package com.iteneum.communitylist.presentation
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.lazy.grid.GridCells
+import androidx.compose.foundation.lazy.grid.LazyVerticalGrid
+import androidx.compose.foundation.lazy.grid.items
+import androidx.compose.runtime.Composable
+import androidx.hilt.navigation.compose.hiltViewModel
+import com.iteneum.designsystem.components.LPGenericElevatedCardImage
+import com.iteneum.designsystem.theme.LeasePertTheme
+import com.iteneum.designsystem.utils.TextUtils.TWO
+
+@Composable
+fun CommunityListView(
+    viewModel: CommunityListViewModel = hiltViewModel()
+) {
+    CommunityListContent(
+        state = viewModel.uiState
+    )
+}
+
+@Composable
+fun CommunityListContent(
+    state: CommunityListUIState
```

Comment:
Please be more descriptive in the name of the variables in this case what represent this state

Reply:
It's going to be deleted I'm going to add it next PR